pkg/sac/scope_checker_or.go

```diff
@@ -0,0 +1,208 @@
+package sac
+
+import (
+	"context"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	"github.com/stackrox/rox/pkg/sac/effectiveaccessscope"
+)
+
+type orScopeChecker struct {
+	scopeCheckers []ScopeChecker
+}
+
+// NewOrScopeChecker returns a new scope checker wrapping the given scope checkers,
+// each result will be combined in a disjunction (OR).
+func NewOrScopeChecker(scopeCheckers ...ScopeChecker) ScopeChecker {
+	return orScopeChecker{
+		scopeCheckers: scopeCheckers,
+	}
+}
+
+func (s orScopeChecker) Core() ScopeCheckerCore {
+	if len(s.scopeCheckers) != 0 {
+		return s.scopeCheckers[0].Core()
+	}
+	return nil
+}
```

Comment:
This is only used within [verifying_interceptor.go](https://github.com/stackrox/stackrox/blob/d653406651df4331a714839ec2c0a23a93425c64/central/sac/transitional/verifying_interceptor.go#L74), I'm not sure how to do this unambiguously here.